azure-pipelines.yml

```diff
@@ -69,6 +69,7 @@ jobs:
       condition: and(succeeded(), not(eq(variables['skip-github'], 'TRUE')))
     - bash: |
         set -euo pipefail
+        eval "$(./dev-env/bin/dade-assist)"
```

Comment:
I think it makes sense to at least add a comment that getting things like gsutil from older dev-envs is slightly sketchy so that if it ever breaks we at least understand why easily.